core/src/main/java/com/rim/IceField/Map.java

```diff
@@ -1,33 +1,106 @@
 package com.rim.IceField;
 
+import java.util.ArrayList;
+
 public class Map {
 
-    //public Blizzard blizzard;
-      private Iceberg iceberg;
-      private PlayerBase playerBase;
-      //public Inventory inventory;
+      //private Blizzard blizzard;
+      private ArrayList<Iceberg> icebergs;
+      private ArrayList<PlayerBase> players;
+      private ArrayList<ItemBase> items;
+
+
+
+    public ArrayList<Iceberg> getIcebergs() {
+        return icebergs;
+    }
 
+    public ArrayList<PlayerBase> getPlayers() {
+        return players;
+    }
 
-    public Iceberg getIceberg() {
-        return iceberg;
+    public ArrayList<ItemBase> getItems() {
+        return items;
     }
 
-    public void setIceberg(Iceberg iceberg) {
-        this.iceberg = iceberg;
+//    public Blizzard getBlizzard(){
+//        return blizzard;
+//    }
+
+    public void setIcebergs(ArrayList<Iceberg> icebergs) {
+        this.icebergs = icebergs;
     }
 
-    public PlayerBase getPlayerBase() {
-        return playerBase;
+    public void setItems(ArrayList<ItemBase> items) {
+        this.items = items;
     }
 
-    public void setPlayerBase(PlayerBase playerBase) {
-        this.playerBase = playerBase;
+    public void setPlayers(ArrayList<PlayerBase> players) {
+        this.players = players;
     }
 
+//    public void setBlizzard(Blizzard blizzard) {
+//        this.blizzard = blizzard;
+//    }
+
+
+
+    public void generateItemsOnMap(ArrayList<PlayerBase> players){
+
+          // - - - - - - - - - - - - - - - - - - - - - Blizzard - - - - - - - - - - - - - - - - - - - - -
+
+          //this.blizzard = new Blizzard();
+          //blizzard.Blow();
+
+         // - - - - - - - - - - - - - - - - - - - - - Items - - - - - - - - - - - - - - - - - - - - -
+
```

Comment:
I think that the implementation of the items here should be a little different. I thought that every item belongs to one iceberg or there are icebergs that don't have items and say have just snow.

Reply:
But this is the initialization, and as we are going to do the test-case we have to set them the way we need to show the use-case. And in the Iceberg initialization we set the items , or leave some of them emty. So I don't get your point here.

Reply:
I think that some items should be assigned to the icebergs